### PR TITLE
Components

### PR DESCRIPTION
**Pull request by Fahim-Khan-P**
### **I've added the following requirements!😎😎😉**

- [ ] Use class based components (All components should be class-based).
- [ ] Once the Calculator is ready, import it and use it in your main component (App.js).
- [ ] Delete all😃 the boilerplate from CRA (text, images, styles).
- [ ] Create a directory called components.
- [ ] Inside components, create a new Calculator.js file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,126 @@
-# Math-magician
\ No newline at end of file
+<a name="readme-top"></a>
+<div align="center">
+
+  <img src="https://github.com/microverseinc/readme-template/blob/master/murple_logo.png" alt="logo" width="140"  height="auto" />
```
Comment (t-yanick) [inline README.md]: - [ ] Kindly remove the template logo. You can replace with a personal logo if you have one.

File: src/App.css [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  width: 50%;
+  grid-row-gap: 12px;
+  border-radius: 30px;
+}
+
+.displaySection {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
```
Comment (t-yanick) [inline src/App.css]: - [ ] Kindly follow the design given in the project description. Following the given design is very important as you will be working for clients and will be expected to deliver exactly what they demand or desire.
- Given UI with project requirements
image
- Your design
image
- The gaps between the buttons needs to be fixed
- The background color where the answer is displayed needs the color fixed
- When the calculator is switched ON, the 0 should be to the RIGHT as on the design UI given.
- The button colors needs to be exact
- I don't think you need the border radius if you have to respect the given UI (For both the calculator window and the buttons)

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @Fahim-Khan-P , 

### Good job so far!

### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by roshan-bajgain — APPROVED** [review]
Hi @Fahim-Khan-P, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**